kw

```diff
@@ -287,6 +287,13 @@ function kw()
         vm_main "$@"
       )
       ;;
+    handle-trailer)
```

Comment:
This name is kind of bad in fact. We couldn't come up with a better name while making this feature.

Not sure if this will be put into another already existing feature in KW or if it's going to be a different one. If the latter, then a better name is welcome.

We are open to suggestions.